mpqp/noise/noise_model.py

```diff
@@ -344,6 +344,8 @@ def __init__(
             raise ValueError(
                 f"Invalid probability: {prob} but should be between 0 and 0.5"
             )
+        if targets is None:
```

Comment:
shouldn't this be dealt with and the `NoiseModel` level ?

Reply:
done